dev-python/spyder/spyder-2.3.8.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+#
+EAPI=5
```

Comment:
I thought EAPI should be defined from 0 to 5 ([https://wiki.gentoo.org/wiki/EAPI](url))... Should I change it do 6? What improvements would this bring?